registry/unit.go

```diff
@@ -61,8 +62,48 @@ func (r *EtcdRegistry) getUnitByHash(hash unit.Hash) *unit.UnitFile {
 		}
 		return nil
 	}
+	return r.unitFromEtcdNode(hash, resp.Node)
+}
+
+// getAllUnitsHashMap retrieves from the Registry all Units and returns a map of hash to Unit
+func (r *EtcdRegistry) getAllUnitsHashMap() map[string]*unit.UnitFile {
```

Comment:
@miekg: the naming convention is consistent in the file
@bcwaldon: added return